Templates/APIs.mustache

```diff
@@ -13,15 +13,15 @@ public class API {}
 /// Helpers for update the headers in `API.Config`
 extension API.Config {
 
-    func append(headers: [String: String]) -> API.Config {
+    public func append(headers: [String: String]) -> API.Config {
```

Comment:
We should rename these functions to `appending` to keep up with Apple conventions.